odk1-src/form-logic.rst

```diff
@@ -0,0 +1,314 @@
+***********
+Form Logic
```

Comment:
How about notifying the user to refer http://xlsform.org first?

Reply:
Agreed! The form logic section is currently tightly focused on XLSForm, so we should say that up front. We have a similar language in form-widgets that we can use.